11/umbraco-cms/fundamentals/backoffice/settings.md

```diff
@@ -0,0 +1,83 @@
+
+</details>
+
+<details>
+
+<summary>Published Status</summary>
+
+The Published Status dashboard displays the status of your site in the Published Cache Status section alongwith the Content and Media nodes value. The Caches section provides three options: Memory Cache, Database Cache, and Internals (NuCache).
+
+- Memory Cache - Reloads the in-memory cache by entirely reloading it from the database cache. Use it when you think that the memory cache has not been properly refreshed.
+
+- Database Cache - Rebuilds the database cache that is the content of the `cmsContentNu` table. Use it when reloading the Memory Cache is not enough and you think that the database cache has not been properly generated.
+
+- Internals - Lets you trigger a NuCache snapshots collection.
+
+</details>
+
+<details>
+
+<summary>Models Builder</summary>
+
+Models builder is a tool that can generate a complete set of strongly-typed published content models for Umbraco. Models are available in both controllers and views.
+
+For more information about Models Builder, see the [Models Builder](../reference/templating/modelsbuilder/README.md) article.
+
+</details>
+
+<details>
+
+<summary>Health Check</summary>
+
+Health Checks are used to determine the status of your Umbraco project. It is a handy list of checks to see if your Umbraco installation is configured according to best practices. It's possible to add your custom-built health checks.
+
+For more information about Health Checks, see the [Health Check](health-check/README.md) article.
+
+</details>
+
+<details>
+
+<summary>Profiling</summary>
+
+You can use the built-in performance profiler to assess the performance when rendering pages. To activate the profiler for a specific page rendering, add `umbDebug=true` to the querystring when requesting the page.
+
+The Profiling dashboard provides a toggle option - `Activate the profiler by default` to keep the profiler active by default for all page renderings. You can use this option without having to set `umbDebug=true` on each page request. The toggle button sets a cookie named `UMB-DEBUG` in your browser, which then activates the profiler automatically.
+
+For more information about MiniProfiler, see the [MiniProfiler](../code/debugging/README.md#miniprofiler) section in the [Debugging](../code/debugging/README.md) article.
+
+</details>
+
+<details>
+
+<summary>Telemetry Data</summary>
+
+The Telemetry Data dashboard is a consent screen that is used for collecting system and usage information from your installation. Here, you can see what type of data is being collected and even adjust the level of reporting. Currently, there are three levels available: **Minimal**, **Basic**, and **Detailed**. **Basic** is the default option where the data sent is an anonymized site ID, Umbraco version, and packages installed. You can see the specific data being sent on each of the levels directly in the Dashboard.
```

Comment:
As of the latest version of CMS (in RC currently), the default option is "Detailed".

```suggestion
The Telemetry Data dashboard is a consent screen that is used for collecting system and usage information from your installation. Here, you can see what type of data is being collected and even adjust the level of reporting. Currently, there are three levels available: **Minimal**, **Basic**, and **Detailed**. **Basic** is the default option where the data sent is an anonymized site ID, Umbraco version, and packages installed. You can see the specific data being sent on each of the levels directly in the Dashboard.
```